css/style.css

```
@@ -1,8 +1,22 @@
 .container {
+    background-color: #eddcd2;
 
 }
+.main {
+    padding: 3px;
+    background-color: #fff1e6;
+}
 .checkBox {
     margin: 0px 8px 0px 8px;
     text-align: center;
 
+}
+.title {
+    background-color: #b7b7a4;
+    text-align: center;
+    border-radius: 5px;
+    margin: 3px;
+    margin-top: 5px;
+    color: #362c28;
```

The dark text on the darker background results in the title not standing out as much as the rest of the page. Perhaps making the text color ligher would help. Alternatively, the title background could be made lighter.
```suggestion
    color: white;
```

---

I don't really prefer the brown background but you were probably adding that in as a placeholder Everett, we can review this later tonight as a group.